src/REPLICA/fix_neb.cpp

```diff
@@ -62,22 +62,29 @@ FixNEB::FixNEB(LAMMPS *lmp, int narg, char **arg) :
   // optional params
 
   NEBLongRange = false;
+  EqualForceNEB = false;
   StandardNEB = true;
+  
   PerpSpring = FreeEndIni = FreeEndFinal = false;
   FreeEndFinalWithRespToEIni = FinalAndInterWithRespToEIni = false;
   kspringPerp = 0.0;
   kspringIni = 1.0;
   kspringFinal = 1.0;
-
   int iarg = 4;
   while (iarg < narg) {
     if (strcmp(arg[iarg],"parallel") == 0) {
       if (iarg+2 > narg) error->all(FLERR,"Illegal fix neb command");
       if (strcmp(arg[iarg+1],"ideal") == 0) {
         NEBLongRange = true;
+        EqualForceNEB = false;
```

Comment:
Rather than having a growing list of booleans with constraints of which may be true or false, we should convert this to have just one integer variable that selects the "NEB mode" and then an enumerator with symbolic constants to choose one of the available ones. This would make it more straightforward to add a fourth or fifth, if that need ever arises.

Reply:
Agreed, I avoided this to change the existing code as little as possible. I can implement this

Reply:
I suggest to have the constants reflect the name of the keywords. Something like:
```
enum {NEIGHBOR, IDEAL, EQUAL};
```